### PR TITLE
feat: Accept invite flow

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res [hunk 1]
```diff
@@ -22,6 +22,14 @@ module TermsAndCondition = {
   }
 }
 
+let flowTypeStrToVariantMapper = val => {
+  switch val {
+  | "merchant_select" => MERCHANT_SELECT
+  | "dashboard_entry" => DASHBOARD_ENTRY
+  | _ => MERCHANT_SELECT
```
Comment (JeevaRamu0104) [inline src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res:29]: set default value as unknown

File: config/FeatureFlag.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "test_live_toggle": false,
   "is_live_mode": false,
-  "magic_link": false,
+  "magic_link": true,
```
Comment (Riddhiagrawal001) [inline config/FeatureFlag.json:4]: remove feature flag changes

File: src/entryPoints/hyperswitch/AcceptInvite.res [hunk 3]
```diff
@@ -0,0 +1,139 @@
+@react.component
+let make = () => {
+  open HSwitchUtils
+  open APIUtils
+  open LogicUtils
+
+  let updateDetails = useUpdateMethod()
+  let merchantDataFromLocalStorage =
+    LocalStorage.getItem("accept_invite_data")
```
Comment (Riddhiagrawal001) [inline src/entryPoints/hyperswitch/AcceptInvite.res:9]: Can we use `getInfoFromLocalStorage` func to fetch the value of accept_invite_data

File: src/entryPoints/hyperswitch/AcceptInvite.res [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        }
+        acc
+      })
+      let body =
+        [
+          ("merchant_ids", acceptedMerchantIds->JSON.Encode.array),
+          ("need_dashboard_entry_response", true->JSON.Encode.bool),
+        ]
+        ->Dict.fromArray
+        ->JSON.Encode.object
```
Comment (Riddhiagrawal001) [inline src/entryPoints/hyperswitch/AcceptInvite.res:47]: `LogicUtils. getJsonFromArrayOfJson`

File: src/entryPoints/hyperswitch/AcceptInvite.res [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+      merchantData
+      ->JSON.stringifyAny
+      ->Option.getOr("")
+      ->safeParse
+      ->JSON.Decode.array
+      ->Option.getOr([])
+
+    merchantDataUpdated
+    ->Array.get(index)
+    ->Option.getOr(JSON.Encode.null)
```
Comment (Riddhiagrawal001) [inline src/entryPoints/hyperswitch/AcceptInvite.res:70]: `LogicUtils.getValueFromArray `

File: src/entryPoints/hyperswitch/AcceptInvite.res [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    <div className="h-full w-full flex items-center justify-center">
+      <div className="bg-white h-35-rem w-200 rounded-2xl">
+        <div className="p-6 border-b-2">
+          <img src={`assets/Dark/hyperswitchLogoIconWithText.svg`} />
+        </div>
+        <div className="p-6 flex flex-col gap-2">
+          <p className={`${textHeadingClass} text-grey-900`}>
+            {"Hey there, welcome to Hyperswitch!"->React.string}
+          </p>
+          <p className={`${textSubHeadingClass}`}>
```
Comment (Riddhiagrawal001) [inline src/entryPoints/hyperswitch/AcceptInvite.res:87]: p className=textSubHeadingClass

File: src/entryPoints/hyperswitch/AcceptInvite.res (hunk 4 above)
Comment (PritishBudhiraja) [reply src/entryPoints/hyperswitch/AcceptInvite.res:47]: Updated

File: src/entryPoints/hyperswitch/AcceptInvite.res (hunk 5 above)
Comment (PritishBudhiraja) [reply src/entryPoints/hyperswitch/AcceptInvite.res:70]: Updated

File: src/entryPoints/hyperswitch/AcceptInvite.res (hunk 6 above)
Comment (PritishBudhiraja) [reply src/entryPoints/hyperswitch/AcceptInvite.res:87]: Updated

File: src/entryPoints/hyperswitch/AcceptInvite.res (hunk 3 above)
Comment (PritishBudhiraja) [reply src/entryPoints/hyperswitch/AcceptInvite.res:9]: Not needed it's only required in this file

File: config/FeatureFlag.json (hunk 2 above)
Comment (PritishBudhiraja) [reply config/FeatureFlag.json]: Updated

File: src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res (hunk 1 above)
Comment (PritishBudhiraja) [reply src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res]: Updated

File: src/screens/HyperSwitch/Routing/RoutingUtils.res [hunk 7]
```diff
@@ -225,7 +224,7 @@ module ConfigureRuleButton = {
 
 let validateNameAndDescription = (~dict, ~errors) => {
   ["name", "description"]->Array.forEach(field => {
-    if dict->LogicUtils.getString(field, "")->String.trim === "" {
+    if dict->getString(field, "")->String.trim === "" {
```
Comment (Riddhiagrawal001) [inline src/screens/HyperSwitch/Routing/RoutingUtils.res:227]: String .length

**Review by Riddhiagrawal001 — DISMISSED** [review]

File: src/screens/HyperSwitch/Routing/RoutingUtils.res (hunk 7 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Routing/RoutingUtils.res:227]: Updated

**Review by Riddhiagrawal001 — DISMISSED** [review]

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**